graylog2-server/src/main/java/org/graylog2/system/stats/elasticsearch/ElasticsearchProbe.java

```diff
@@ -17,80 +17,116 @@
 package org.graylog2.system.stats.elasticsearch;
 
 import com.google.common.collect.Lists;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
 import com.google.inject.Singleton;
-import org.elasticsearch.action.admin.cluster.health.ClusterHealthRequest;
-import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse;
-import org.elasticsearch.action.admin.cluster.stats.ClusterStatsNodes;
-import org.elasticsearch.action.admin.cluster.stats.ClusterStatsRequest;
-import org.elasticsearch.action.admin.cluster.stats.ClusterStatsResponse;
-import org.elasticsearch.action.admin.cluster.tasks.PendingClusterTasksRequest;
-import org.elasticsearch.action.admin.cluster.tasks.PendingClusterTasksResponse;
-import org.elasticsearch.client.Client;
-import org.elasticsearch.client.ClusterAdminClient;
-import org.elasticsearch.cluster.service.PendingClusterTask;
+import io.searchbox.client.JestClient;
+import io.searchbox.client.JestResult;
+import io.searchbox.cluster.Health;
+import io.searchbox.cluster.PendingClusterTasks;
+import io.searchbox.cluster.Stats;
 import org.graylog2.indexer.IndexSetRegistry;
 
 import javax.inject.Inject;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Locale;
 
 @Singleton
 public class ElasticsearchProbe {
-    private final Client client;
+    private final JestClient jestClient;
     private final IndexSetRegistry indexSetRegistry;
 
     @Inject
-    public ElasticsearchProbe(Client client, IndexSetRegistry indexSetRegistry) {
-        this.client = client;
+    public ElasticsearchProbe(JestClient jestClient, IndexSetRegistry indexSetRegistry) {
+        this.jestClient = jestClient;
         this.indexSetRegistry = indexSetRegistry;
     }
 
     public ElasticsearchStats elasticsearchStats() {
-        final ClusterAdminClient adminClient = client.admin().cluster();
+        final JestResult clusterStatsResponse;
+        try {
+            clusterStatsResponse = jestClient.execute(new Stats.Builder().build());
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
```

Comment:
Maybe we could wrap it in a (subclass of the) `ElasticsearchException`? This makes it a bit easier to catch this specific case on higher levels.